scripts/presubmit.sh

```diff
@@ -37,28 +37,21 @@ fi
 
 
 echo "📚 Fetch dependencies"
-OUT="$(go get -t -tags=performance,e2e ./... 2>&1)" || {
+OUT="$(go get -t ./... 2>&1)" || {
   echo "✋ Error fetching dependencies"
   echo "\n\n${OUT}\n\n"
   exit 1
 }
 
 
 echo "📚 Fetch test dependencies"
-OUT="$(go test -i -tags=performance,e2e ./... 2>&1)" || {
+OUT="$(go test -i ./... 2>&1)" || {
   echo "✋ Error fetching test dependencies"
   echo "\n\n${OUT}\n\n"
   exit 1
 }
 
 
-echo "↹ Verify tabs"
```

Comment:
should we preserve all of these checks as client side executable (can be different script)?

Reply:
They already are 😄 

The CI instruction is `make tabcheck` which still exists and can be run locally. Same thing with `make lint`. It's a personal pet peeve of mine to have to rely on a CI system to get feedback.